nbgrader/converters/base.py

```diff
@@ -248,6 +260,24 @@ def set_permissions(self, assignment_id, student_id):
         for dirname, _, filenames in os.walk(dest):
             for filename in filenames:
                 os.chmod(os.path.join(dirname, filename), permissions)
+            # If groupshared, set dir permissions - see comment below.
+            if self.groupshared and os.stat(dirname).st_uid == os.getuid():
+                #for subdirname in subdirnames:
+                #    subdirname = os.path.join(dirname, subdirname)
+                try:   os.chmod(dirname, (os.stat(dirname).st_mode|0o2770) & 0o2777)
```

Comment:
This is a style nitpick, but would you mind formatting the try/except statements so they are across 4+ lines, like:

```
try:
    ...
except PermissionError:
    ...
```

so that it matches the style of the rest of the code?